### DISCUSSION TITLE
Starting multiple processes within multiple processes in Python

### DISCUSSION THREAD

**Discussion by YYH211** [fastapi/fastapi#12938] [Questions]
### Privileged issue

- [x] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

I defined a multiprocessing pool in the initialization function, and then started multiple processes within a single process, resulting in an error.：：AssertionError: daemonic processes are not allowed to have children
The code works fine outside of your framework, so is it due to some limitations in your framework or another reason?

**Comment by YuriiMotov** [comment]
Closed it for now to keep discussions clean.
If you still need help, please provide MRE and re-open this topic
